src/Surfnet/StepupMiddleware/CommandHandlingBundle/Configuration/Command/CreateInstitutionConfigurationCommand.php

```diff
@@ -19,8 +19,9 @@
 namespace Surfnet\StepupMiddleware\CommandHandlingBundle\Configuration\Command;
 
 use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\AbstractCommand;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\Command\ManagementExecutable;
 
-class CreateInstitutionConfigurationCommand extends AbstractCommand
+class CreateInstitutionConfigurationCommand extends AbstractCommand implements ManagementExecutable
```

Comment:
is that so? What is the current user if a new Identity leads to the discovery of a new Institution?